app.js

```diff
@@ -14,22 +14,38 @@ function create(env, ctx) {
     var appInfo = env.name + ' ' + env.version;
     app.set('title', appInfo);
     app.enable('trust proxy'); // Allows req.secure test on heroku https connections.
-    if (process.env.INSECURE_USE_HTTP !== 'true') {
+    if (!env.settings.isEnabled('insecureUseHttp')) {
         app.use((req, res, next) => {
         if (req.header('x-forwarded-proto') !== 'https')
-            res.redirect(`https://${req.header('host')}${req.url}`)
+            res.redirect(`https://${req.header('host')}${req.url}`);
         else
             next()
         })
-        if (process.env.SECURE_HTTP_HEADERS == 'true') {
-            const helmet = require('helmet')
+        //if (env.settings.isEnabled('secureHstsHeader')) { // Add HSTS (HTTP Strict Transport Security) header by
```

Comment:
This syntax does not seem to work for some reason.

Reply:
@PieterGit env.settings.isEnabled didn't work for me either so that's why I introduced the process.env construct instead. Speaking of which, I don't think CORS works as expected with the current env.settings.isEnabled construct but it's not something I use so I am not the right person to test the functionality. Are we aware of users that uses CORS today?